go.mod

```diff
@@ -19,11 +19,11 @@ require (
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
There are 2 dependencies that can probably be removed from the `replace` section as they are already defined here in the `require` section:
```
k8s.io/api v0.18.6
k8s.io/apimachinery v0.18.6
```

Reply:
Good catch! I will submit a follow up PR to clean that up.

Reply:
Good to know. I don't have a huge grasp on the intricacies of Go dependency management just yet.